dart-common/src/main/java/com/f2prateek/dart/common/AbstractDartProcessor.java

```diff
@@ -163,6 +167,73 @@ protected InjectionTarget getOrCreateTargetClass(Map<TypeElement, InjectionTarge
     return injectionTarget;
   }
 
+  protected void findParentInjectionTarget(TypeElement typeElement,
+                                                                InjectionTarget injectionTarget) {
+    findInjectionTargetForSuperTypeWithInjectExtra(typeElement,
+            injectionTarget,
+            false,
+            true);
+  }
+
+  protected void findAllParentsInjectionMap(TypeElement typeElement,
```

Comment:
name a little misleading. It finding the parent and inheriting the injections from all ancestors.